app/mailers/audit_mailer.rb

```diff
@@ -0,0 +1,21 @@
+class AuditMailer < ApplicationMailer
+  helper :audits, :layout
+
+  def summary(options = {})
+    raise ::Foreman::Exception.new(N_("Must specify a user with email enabled")) unless (user=User.find(options[:user])) && user.mail_enabled?
+    time     = options[:time] ? "time > \"#{options[:time]}\"" : 'time > yesterday'
+    @query    = "#{options[:query]} and #{time}"
```

Comment:
I'm not certain we should rely on that behaviour, try and make it a well-formed search term if possible!